### ISSUE TITLE
WCAG references in web/dev

### ISSUE THREAD

**Issue by Aniort** [Orange-OpenSource/a11y-guidelines#858]
We need to review/update WCAG references inside Web/Dev criteria, some are, at least, missing (form)

4 days to compmete this task

**Comment by jeromebriard** [comment]
Note : 
Page: [https://a11y-guidelines.orange.com/en/mobile/android/development/vocal-navigation/] (English version); WCAG references are noted by the language code “en”. This tag should be removed, as the page is in English.
